doc/writeTask.dox

```diff
@@ -230,5 +232,65 @@ page \ref LSST_task_documentation, even if your task is not in the pipe_tasks pa
 ## \}
 \endverbatim
 
+\section pipeTasks_writeTask_subtasks Subtasks
+
+As mentioned above, each subtask is specified in the configuration as a field of type
```

Comment:
I would remove the "As mentioned above" part of the sentence. This creates a textual dependency that might be broken at some point and cause confusion.